components/automate-deployment/pkg/converge/converger.go

```diff
@@ -17,44 +21,38 @@ const (
 	elasticsearchServiceName = "automate-elasticsearch"
 )
 
-type Converger interface {
-	Converge(uint64, *Task, DesiredState, EventSink) error
-	Stop()
-}
+var (
+	waitingForReconfigureTimeout = 30 * time.Second
```

Comment:
seems a bit short. Given that this is an anomoly, maybe we can be more lenient

Reply:
This was the previous timeout (sort of), but I agree we could probably make it longer. I guess the trade-off is that the longer we stay in this state the longer we are prevented from doing anything else.  That said, considering that we've seen production nodes where hab starts taking 30 seconds per command, perhaps we can bump this to at least 1 minute?

Reply:
We should revert this change I think.  It turns out that we hit this case nearly all of the time because reconfiguration rarely leads to a template being rendered in the case of the deployment service.  I'll change this back to 30s and we can follow up with a change that removes some of our most confusing reconfiguration logic.